dev/docker/doris/start.sh

```diff
@@ -30,6 +30,9 @@ DORIS_BE_SCRIPT="${DORIS_BE_HOME}/bin/start_be.sh"
 sed -i '/Please set vm.max_map_count/,/exit 1/{s/exit 1/#exit 1\n        echo "skip this"/}' ${DORIS_BE_SCRIPT}
 sed -i '/Please set the maximum number of open file descriptors/,/exit 1/{s/exit 1/#exit 1\n        echo "skip this"/}' ${DORIS_BE_SCRIPT}
 
+# remove chmod command in start_be.sh, it will cost a lot of time. Add it to Dockerfile
+sed -i 's/chmod 755 "${DORIS_HOME}\/lib\/doris_be"/#&/' ${DORIS_BE_SCRIPT}
```

Comment:
Why is the operation taking so much time? Is the file count too large or does its size take up too much space?

Reply:
This file is a single file, the size is 3.2GB.  We can guess from log, it costs 10+ seconds in chmod.

In the FrontEnd IT Test, it takes more time (up to 30 seconds). I believe it may be due to disk performance.

Reply:
I see.